app/settings_test.go

```diff
@@ -5,11 +5,21 @@ import (
 	"path/filepath"
 	"testing"
 
+	"fyne.io/fyne"
 	"github.com/stretchr/testify/assert"
 
+	"fyne.io/fyne/test"
 	"fyne.io/fyne/theme"
 )
 
+func TestSettingsBuildType(t *testing.T) {
+	set := test.NewApp().Settings()
+	assert.Equal(t, fyne.StandardBuild, set.BuildType()) // during test we should have a normal build
+
+	set = &settings{}
+	assert.Equal(t, buildMode, set.BuildType()) // when testing this package only it could be debug or release
```

Comment:
Not sure I follow the comment here. The assertion seems to just test the getter returns the constant.

Reply:
It is attempting to demonstrate / document the difference between the BuildType API which must always be StandardBuild for testing, but the tests in this package are testing the internal settings code.
I considered not adding the second line, but it felt like it could be misleading for anyone who wanted to use the buildMode constant during test

Reply:
Hmm so line 17 should always pass as testApp.BuildType() always returns Standard, and line 20 tests that a generic settings should return whatever the package was built as (Standard, Debug, or Release)